modules/ROOT/pages/backup-restore/restore-backup.adoc

```diff
@@ -116,18 +111,108 @@ Usage of this option is only allowed if the `--from-path` parameter points to on
 |===
 
 [[restore-backup-example]]
-== Example
+== Examples
 
-The following is an example of how to perform an online restore of the database backup created in the section xref:backup-restore/online-backup.adoc#online-backup-example[Back up an online database], using the `neo4j-admin database restore` command.
+.Restore a database backup
+====
 
+. Restore a database backup created in the section xref:backup-restore/online-backup.adoc#online-backup-example[Back up an online database]. The backup artifact is located in the `/path/to/mybackups` directory.
```

Comment:
Multiple times when talking to people who have used the restore command I felt that the fact that we need to pass the last backup of a chain was not well understood. Perhaps we should repeat again that information and maybe use the same table as below to illustrate:

```
|                                                   FILE | DATABASE |                          DATABASE ID |                TIME |  FULL | COMPRESSED | LOWEST TX | HIGHEST TX |
| file:///path/to/mybackups/neo4j-2023-06-29T14-46-27.backup |    neo4j | c8368b24-55e2-474d-bb41-75657f5bfcde | 2023-06-29T13:46:27 |  true |       true |         1 |         11 |
| file:///path/to/mybackups/neo4j-2023-06-29T14-50-45.backup |    neo4j | c8368b24-55e2-474d-bb41-75657f5bfcde | 2023-06-29T13:50:45 | false |       true |        12 |         14 |
| file:///path/to/mybackups/neo4j-2023-06-29T14-51-33.backup |    neo4j | c8368b24-55e2-474d-bb41-75657f5bfcde | 2023-06-29T13:51:33 | false |       true |        15 |         18 |
```

saying for instance: "If you restore `neo4j-2023-06-29T14-50-45.backup` your db will have 14 as last tx id"